draft-ietf-quic-transport.md

```diff
@@ -6706,21 +6709,22 @@ An off-path attacker can:
 
 An off-path attacker cannot:
 
-- Modify any part of a packet
+- Modify packets sent by endpoints
 - Delay packets
 - Drop packets
 - Reorder original packets
 
-An off-path attacker can modify packets that it has observed and inject them
-back into the network, potentially with spoofed source and destination
-addresses.
+An off-path attacker can create modified copies of packets that it has observed
+and inject those copies into the network, potentially with spoofed source and
+destination addresses.
 
 For the purposes of this discussion, it is assumed that an off-path attacker
-has the ability to observe, modify, and re-inject a packet into the network
-that will reach the destination endpoint prior to the arrival of the original
-packet observed by the attacker. In other words, an attacker has the ability to
-consistently "win" a race with the legitimate packets between the endpoints,
-potentially causing the original packet to be ignored by the recipient.
+has the ability to observe and re-inject a modified copy of a packet into the
```

Comment:
```suggestion
has the ability to inject a modified copy of a packet into the
```
Given that we've just talked about the observe-modify-reinject process in the previous sentence, I think we're safe to shorthand this to injecting a modified packet.